resources/js/Directives/languagesDirective.js

```diff
@@ -20,15 +20,15 @@ app.directive('languages',  ['$http',  '$timeout' ,function ($http, $timeout) {
                 $scope.lang_loader=true;
                 $http.get(api_link + '/list/bears')
                     .then(function(data){
-                        arr = []
-                        angular.forEach(Object.keys(data["data"]), function(value, key){
+                        arr = [];
```

Comment:
This ';' is unrelated to the changes. https://github.com/coala/landing-frontend/pull/177#discussion_r122586704 was about that is not part of this problem.
@gkrishnan724, if you want changes to existing code, please create a new issue.